website/docs/client/java/configuration.md

```diff
@@ -41,27 +41,10 @@ transport:
 facets:
   spark_unknown:
     disabled: true
-  spark:
-    logicalPlan:
+  "spark.logicalPlan":
       disabled: true
 ```
 
-### Deprecated syntax
```

Comment:
Can we keep that section, just reworded to make clear that those configurations were removed?

Reply:
yes we can.